### DISCUSSION TITLE
Error when i try to connect 1.19.2 server

### DISCUSSION THREAD

**Discussion by tomhubert50400** [PrismarineJS/mineflayer#3094] [Q&A]
Here's the error : 

kicked {"extra":[{"color":"red","text":"Not authenticated with "},{"color":"red","clickEvent":{"action":"open_url","value":"http://Minecraft.net"},"text":"Minecraft.net"},{"color":"red","text":""}],"text":""}
Déconnecté du serveur Minecraft

Here's my code : 

const mineflayer = require('mineflayer');

const bot = mineflayer.createBot({
  host: 'PLAY.FRENCHSKY.NET', // Adresse du serveur Minecraft
  auth:'microsoft',
  username: 'Sample', // Nom d'utilisateur de votre compte Minecraft
  password: 'Sample', // Mot de passe de votre compte Minecraft (si applicable)
  version:'1.19.2'
});

bot.once('spawn', () => {
  // Lorsque le bot est connecté et apparaît dans le jeu
  console.log('Bot connecté au serveur Minecraft');
});

// Événement de chat
bot.on('chat', (username, message) => {
  // Afficher le message du chat dans la console
  console.log(`${username}: ${message}`);
});

bot.on('end', () => {
    console.log('Déconnecté du serveur Minecraft');
    // Effectuez ici les actions nécessaires en cas de déconnexion
  });
  
bot.on('error', (err) => {
    console.log('Erreur de connexion au serveur Minecraft:', err);
    // Effectuez ici les actions nécessaires en cas d'erreur
  });

  bot.on("kicked", reason => console.error("kicked", reason));

My credentials are good, if anyone has an idea, as a beginner i need help

thanks

**Comment by PondWader** [comment]
Try removing the password field in createBot, then you will be prompted to login manually which is the recommended way.
